backend/blueprint/api/controllers/fileUploader.py

```diff
@@ -37,16 +37,25 @@ def uploadImage():
 
 
 @router.route('/file', methods=['POST'])
-@loginRequired
 def fileUpload():
-    if 'file' not in request.files:
+    try:
+        data = request.json['csvFile']
```

Comment:
Is it alright?

Reply:
Yes @djmgit because the ember sends data following the JSON API spec and before sending the data i have serialized the same in this format because of the name of the model. You can verify it from the video in the PR description.